pkg/asset/installconfig/gcp/mock/usertags_mock.go

```diff
@@ -0,0 +1,83 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
This code will be merged with mock code in `pkg/asset/installconfig/gcp/mock/gcpclient_generated.go` after updating `google.golang.org/api/cloudresourcemanager` from `v1` to `v3` in the follow up PR.